readme.txt

```diff
@@ -191,6 +191,10 @@ Our Premium Plugins:
 
 == Changelog ==
 
+= [4.2.2] unreleased =
+
+* Tweak - add a period to the ticket header image setting. [44797]
```

Comment:
Capitalize the first letter and remove the period to match the other entries below?